src/dalit-history-month/dalit-history-month.css

```
@@ -0,0 +1,398 @@
+
+    .db-choices.show,
+    .db-continue.show {
+        opacity: 1;
+        transform: translateY(0);
+    }
+    .db-continue {
+        background-color: rgba(255, 255, 255, 0);
+        border: none;
+    }
+}
+@keyframes fadeIn {
```

Stylelint (17.11.0)

[error] Expected keyframe name "fadeIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)